### DISCUSSION TITLE
feature

### DISCUSSION THREAD

**Discussion by TeaqariaWTF** [robertodoering/harpy#727] [Ideas]
I wonder if an update is possible where we can use our own private API.
In this way, I believe that the application will work more efficiently.
![IMG_20220720_003553_988](https://user-images.githubusercontent.com/90970076/179853041-c58d122c-6f4c-4263-95c3-d5e32cd058bc.jpg)

**Comment by robertodoering** [comment]
Yes, this is planned: https://github.com/robertodoering/harpy/issues/718

**Reply by TeaqariaWTF** [reply]
I'm sorry, is it possible to have a special pricing for Turkey?  The current price is too high
![Screenshot_20220720-005704_Google Play Store](https://user-images.githubusercontent.com/90970076/179855665-527dcd20-f9f0-48da-ae17-f0247cf1057c.png)

**Reply by ghost** [reply]
You can try to compile the app yourself, it's free and use the pro features

**Reply by TeaqariaWTF** [reply]
I do not have a computer and do not have the necessary experience
